### DISCUSSION TITLE
REI bug with fabric 1.19

### DISCUSSION THREAD

**Discussion by Wartux61** [shedaniel/RoughlyEnoughItems#976] [Q & A] [answered]
Hi,

I use REI on a fabric 1.19 server and from time to time, when looking for crafts for example, our inventory crashes or our game crashes. We don't know where it can come from, I'll give you the crash report.

Thank you for your help.

-> https://pastebin.com/Jqs9v9q3

**Comment by shedaniel** [comment, marked as answer]
This should be caused by indium.
